src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -39,12 +41,13 @@ type TrackPanelListItemProps = {
   trackStatus: ImageButtonStatus;
   defaultTrackStatus: ImageButtonStatus;
   track: EnsObjectTrack;
+  highlightedTrack: string;
```

Comment:
Notice how a `track` the line above is an object, but the `highlightedTrack` is a string? That doesn't look right. Isn't it highlightedTrackId (or name, or whatever this string is) then?

Reply:
That's true. I've renamed it to `highlightedTrackId` and all the update/change functions.